tests/cmdline/commands/test_computer.py

```diff
@@ -605,6 +605,52 @@ def test_computer_relabel(self):
         # The new label should be available
         orm.Computer.collection.get(label='comp_cli_test_computer')
 
+    def test_computer_delete_with_nodes(self):
+        """check if 'verdi computer delete' works when there are associated nodes"""
+        from aiida.common.exceptions import NotExistent
+
+        label = 'computer_69'
+        compute_temp = orm.Computer(
+            label=label,
+            hostname='localhost',
+            transport_type='core.local',
+            scheduler_type='core.direct',
+            workdir='/tmp/aiida',
+        )
+        compute_temp.store()
+        options = ['core.local', label, '--non-interactive', '--safe-interval', '0']
+        self.cli_runner(computer_configure, options)
```

Comment:
This is simpler, no need to go through the CLI
```suggestion
        computer.configure(safe_interval=0)
```